libs/SalesforceSDKCore/SalesforceSDKCore/Classes/OAuth/SFOAuthCoordinator.m

```diff
@@ -886,6 +886,7 @@ - (void)webView:(UIWebView *)webView didFailLoadWithError:(NSError *)error {
     
     NSURL *requestUrl = [webView.request URL];
     NSString *errorUrlString = [NSString stringWithFormat:@"%@://%@%@", [requestUrl scheme], [requestUrl host], [requestUrl relativePath]];
+    [self.delegate oauthCoordinator:self didBeginAuthenticationWithView:self.view];
```

Comment:
Showing the auth container in the failure use case, so that the user can change the login host.